_docs_integrate/version-5.md

```diff
@@ -99,6 +99,69 @@ In addition, in contrast to the RequestItems, a RequestItemGroup did not have to
+
+The `content` of a [Message]({% link _docs_integrate/data-model-overview.md %}#message) must either be a [Mail]({% link _docs_integrate/data-model-overview.md %}#mail), [Notification]({% link _docs_integrate/data-model-overview.md %}#notification), [Request]({% link _docs_integrate/data-model-overview.md %}#request), [ResponseWrapper]({% link _docs_integrate/data-model-overview.md %}#responsewrapper) or an [ArbitraryMessageContent]({% link _docs_integrate/data-model-overview.md %}#arbitrarymessagecontent).
+
+The `content` of a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) must either be a [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshipcreationcontent) or an [ArbitraryRelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#arbitraryrelationshiptemplatecontent).
+
+The `creationContent` of a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) must either be a [RelationshipCreationContent]({% link _docs_integrate/data-model-overview.md %}#relationshipcreationcontent) or an [ArbitraryRelationshipCreationContent]({% link _docs_integrate/data-model-overview.md %}#arbitraryrelationshipcreationcontent).
+
+Those new wrappers all have the type
+
+```jsonc
+{
+  "@type": "Arbitrary<...>Content",
+  "value": any,
+}
+```
+
+Thus, when sending a Message with non-standard content via POST `Messages`, instead of sending the parameter `content : <arbitrary-content>`, you send
+
+```jsonc
+content: {
+  "@type": "ArbitraryMessageContent",
+  "value": <arbitrary-content>
+}
+```
+
+Similarly, when creating a RelationshipTemplate with non-standard content via POST `RelationshipTemplates`, you send the parameter
+
+```jsonc
+content: {
+  "@type": "ArbitraryRelationshipTemplateContent",
+  "value": <arbitrary-content>
+}
+```
+
+and the parameter
+
+```jsonc
+creationContent: {
+  "@type": "ArbitraryRelationshipCreationContent",
+  "value": <arbitrary-content>
+}
+```
+
+for creating a Relationship via POST `Relationship`.
+
+Because previously a JSONWrapper was applied to `content: <arbitrary-content>` before actually sending it and the `content` property of the created Message resp. RelationshipTemplate then was
+
+```jsonc
+content: {
+  "@type": "JSONWrapper",
+  "value": <arbitrary-content>
+},
+```
```

Comment:
I think this is incorrect. JSONWrapper only exists internally and when making a JSONWrapper toJSON we only get the value back.

Reply:
good to know